CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# How to contribute
+
+The NEST simulator is a scientific tool and as such it is never ready and constantly changing to meet the needs of novel neuroscientific endeavors. Here you find the the most important information on how you can contribute to NEST. This document is an excerpt from our [developer space](https://nest.github.io/nest-simulator/), which provides more detailed information.
+
+## Getting started
+
+* Make sure you have a [GitHub account](https://github.com/signup/free)
+* The development workflow is based purely on pull requests. This [article](https://nest.github.io/nest-simulator/development_workflow) gives information on how to work with git and GitHub, if you are new to it.
+
+## Making Changes
+
+* Create a topic branch from NEST master in your fork. Please avoid working directly on the `master` branch.
+* Make commits of logical units.
+* Make sure NEST compiles and has no new warnings.
+* Make sure all tests pass (`make installcheck`).
+* Make sure your code conforms to our [coding guidelines](https://nest.github.io/nest-simulator/coding_guidelines_c++#check-static-analysis-local) (`./extras/check_code_style.sh`)
+
+
+## Code Review
+
+We review each pull request according to our [code review guidelines](https://nest.github.io/nest-simulator/code_review_guidelines):
+
+* In general, the rule is that each pull request needs an OK from the CI platform and at least two reviewers to be merged.
+* For changes labeled “not code” or “minor” (e.g. changes in documentation, fixes for typos, etc.), the release manager can waive the need for code review and just accept the OK from Travis in order to merge the request.
```

Comment:
Shouldn't these be normal quotes (around "not code" and "minor"? They look like some kind of fancy unicode quotes to me that might cause trouble on some systems